src/axom/sidre/examples/sidre_createdatastore.cpp

```diff
@@ -415,6 +415,60 @@ void generate_blueprint(DataStore* ds) {
   // _blueprint_generate_save_end
 }
 
+void generate_blueprint_to_path(DataStore* ds) {
+  // _blueprint_generate_path_start
+  // Conduit needs a specific hierarchy.
+  // For this example, we locate the domain data at a path
+  // that is not the top level of its Sidre heirarchy.
+  DataStore cds;
+  std::string domain_name = "domain";
+  std::string domain_location = "domain_data/level/domains/" + domain_name;
+  std::string mesh_name = "mesh";
```

Comment:
Do we have a test/example where the mesh in not named "mesh"?  
If not, could you please change this name to something else?

Reply:
I've given them some different names now.